src/server/endpoints/covidcast.py

```diff
@@ -49,8 +49,10 @@ def parse_source_signal_pairs() -> List[SourceSignalPair]:
     ds = request.values.get("data_source")
     if ds:
         # old version
-        require_any("signal", "signals")
+        require_any("signal", "signals", empty=True)
```

Comment:
What does empty=True do? Does it mean that the following request would go through?

```
https://api.covidcast.cmu.edu/epidata/covidcast/?data_source=jhu-csse&signal&geo=nation:us&time=day:20220101
```

Reply:
Yes, I think so. So we do not want it to go through right? Cuz I see the geo_value has similar situation below and it has this feature.